src/components/select/template.test.js

```diff
@@ -151,6 +151,40 @@ describe('Select', () => {
     })
   })
 
+  describe('when they include option attributes', () => {
+    it('it renders the option attributes', () => {
```

Comment:
Minor – duplicated 'it', don't need to include it as part of the test name.

Reply:
Removed the duplicate, thanks.